.pipelines/pipeline.user.windows.official.yml

```diff
@@ -16,13 +16,11 @@ build:
     - !!buildcommand
       name: 'Assemble ApplicationInsights-Java JARs'
       command: '.scripts/gradle.cmd'
-      arguments: 'assemble copyLibsToGlobalArtifactsFolder -DisRelease=true -Pai.etw.native.build=release'
+      arguments: 'publishToMavenLocal -DisRelease=true -Pai.etw.native.build=release'
       artifacts:
         - to: 'Artifacts'
           include:
-            - '**/build/libs/*.jar'
-            - '**/build/libs/*-sources.jar'
-            - '**/build/libs/*-javadoc.jar'
-            - '**/build/libs/*.pom'
-          exclude:
-            - 'buildSrc/**/*'
+            - '~/.m2/repository/com/microsoft/azure/applicationinsights-agent/*/*.jar'
+            - '~/.m2/repository/com/microsoft/azure/applicationinsights-agent/*/*-sources.jar'
+            - '~/.m2/repository/com/microsoft/azure/applicationinsights-agent/*/*-javadoc.jar'
+            - '~/.m2/repository/com/microsoft/azure/applicationinsights-agent/*/*.pom'
```

Comment:
@heyams I'm still debugging smoke test failure, but can you take a look at the changes I needed to make to pipelines b/c sources/javadocs/pom are no longer built under build/libs, but rather only built using `publishToMavenLocal` under ~/.m2

Reply:
![image](https://user-images.githubusercontent.com/56097766/167500474-ea0d9606-7e94-4b55-b363-1846ef815939.png)

all these yml(s) need to be updated.

Reply:
release pipeline needs to to be updated too. jarPath is different with this change.

Reply:
jar is still built to same place, just the source/javadoc/pom file no longer get build there

Reply:
looks good to me on the yml.. I will update later once you merge if needed.